server/indexer_cmd.go

```diff
@@ -55,7 +55,10 @@ func NewIndexTxCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			stateStore := sm.NewStore(stateDB)
+
+			stateStore := sm.NewStore(stateDB, sm.StoreOptions{
+				DiscardABCIResponses: false,
```

Comment:
I think that this is correct, but please check this, too.

Reply:
I guess whatever config value is fine here, since it's read-only.

Reply:
Great